esrally/mechanic/launcher.py

```diff
@@ -69,6 +69,8 @@ def start(self):
         for cluster_name, cluster_hosts in all_hosts.items():
             all_client_options = self.cfg.opts("client", "options").all_client_options
             cluster_client_options = all_client_options[cluster_name]
+            # Use retries for telemetry connections: http://elasticsearch-py.readthedocs.io/en/latest/index.html#automatic-retries
```

Comment:
Can you please explain the reason why we're doing this in the comment (e.g. to avoid aborts on long-living connections)?